k8s/spinnaker/README.md

```diff
@@ -10,97 +10,112 @@ If you would like to learn more about Spinnaker, please visit the
-installed in your Cloud Shell and the daemon will be installed in a Pod managed
-by a StatefulSet object.
+installed in the management environment (based on Cloud Shell) included in
+this solution. The daemon will be installed in a Pod managed by a StatefulSet
+object.
 
-[Spin](https://www.spinnaker.io/guides/spin/app/) is Spinnaker's CLI. It will
-be installed in your Cloud Shell.
+[Spin](https://www.spinnaker.io/guides/spin/app/) is Spinnaker's CLI. It is also
+available in the management environment.
 
-The Spinnaker solution will not be exposed to external traffic by default.
-A good initial step is accessing Spinnaker via port forwarding, and a script is
-included for automatically establishing port forwarding. In addition, the
-solution includes guidance and automation for exposing Spinnaker via a secure
-domain behind the [Identity-Aware Proxy](https://cloud.google.com/iap/).
+As a safe default, the Spinnaker instance is not exposed to external traffic, and
+is accessed via port forwarding. Port forwarding can be set up with a single command
+from the management environment. Alternatively, the management environment allos you
+to expose Spinnaker via the [Identity-Aware Proxy](https://cloud.google.com/iap/), using
+a secure domain.
 
 # Installation
 
-## Quick install with Google Cloud Shell
+## Quick install via the management console
 
-You must use [Google Cloud Shell](https://cloud.google.com/shell/) to perform the installation.
+You must use the management environment, based on [Cloud Shell](https://cloud.google.com/shell/),
+to perform the installation.
 
 [![Provision Spinnaker](https://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/click-to-deploy.git&cloudshell_working_dir=k8s/spinnaker/scripts/install&cloudshell_tutorial=provision-spinnaker.md)
 
 ### Prerequisites
 
 #### Set up command line tools
 
-Cloud Shell has all the required tools pre-installed.
+The management environment has all the required tools pre-installed.
 
 #### Create a Google Kubernetes Engine cluster
 
-The Spinnaker installation process will create a new GKE cluster by default. If
-you would prefer to use an existing cluster, you must ensure that it has:
+By default, the installation process will create a new GKE cluster for your Spinnaker
+instance. To use an existing cluster, you must ensure that it has:
+
 * IP aliases enabled (since we are using a
-[hosted Redis](https://cloud.google.com/memorystore/) instance)
+[Cloud Memorystore for Redis](https://cloud.google.com/memorystore/) instance)
 * `Full Cloud Platform` scope for its nodes (if using the default service account)
 
-#### Install the Solution
+#### Install Spinnaker for Google Cloud Platform
+
+When you click on the "Open in Google Cloud Shell" button above, the management
+environment will open and ask you to copy/paste and execute a number of commands.
```

Comment:
s/will open and ask/asks/.

Also, add another sentence: "Click to copy/paste, then press Enter to execute."